Update help docs for SPSE (#72)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-server-ps/sharepoint-server/New-SPWebApplicationExtension.md

## PARAMETERS

### -AdditionalClaimProvider

> Adds a specific claim provider to the defined Web application.

Reviewer: ```suggestion
Adds a specific claim provider to the defined web application.
```

```yaml
Type: SPClaimProviderPipeBind[]
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAnonymousAccess

> Allows anonymous access to the Web application zone.

Reviewer: ```suggestion
Allows anonymous access to the web application zone.
```

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowLegacyEncryption

> Specifies that older SSL and TLS protocol versions and cipher suites are allowed to be used with this IIS web site.

Reviewer: ```suggestion
Specifies that older SSL and TLS protocol versions and cipher suites are allowed to be used with this IIS website.
```

Accept wildcard characters: False
```

### -Certificate

> Specifies the certificate that will be used for the Secure Sockets Layer (SSL) binding of this IIS web site.

Reviewer: ```suggestion
Specifies the certificate that will be used for the Secure Sockets Layer (SSL) binding of this IIS website.
```

This parameter is only valid when used with the SecureSocketsLayer parameter.

```yaml
Type: SPServerCertificatePipeBind
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: True (ByValue)
Accept wildcard characters: False
```

### -HostHeader

> Specifies the host header binding for this IIS web site.
> A host header binding allows multiple IIS web sites to share the same port number.
> Web requests sent to a shared port number are routed to the correct IIS web site based on the value of the HTTP host header sent by the client.

Reviewer: ```suggestion
Specifies the host header binding for this IIS website.
A host header binding allows multiple IIS websites to share the same port number.
Web requests sent to a shared port number are routed to the correct IIS website based on the value of the HTTP host header sent by the client.
```

> If no host header binding is specified, then all web requests sent to this port number will be routed to this IIS web site unless another IIS web site has a host header binding that matches the HTTP host header sent by the client.

Reviewer: ```suggestion
If no host header binding is specified, then all web requests sent to this port number will be routed to this IIS website unless another IIS website has a host header binding that matches the HTTP host header sent by the client.
```

Accept wildcard characters: False
```

### -Identity
Specifies the Web application to extend.

```yaml
Type: SPWebApplicationPipeBind
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: True
Position: 1
Default value: None
Accept pipeline input: True (ByValue)
Accept wildcard characters: False
```

### -Name

> Specifies the name of the new IIS Web site in the Web application.

Reviewer: ```suggestion
Specifies the name of the new IIS website in the web application.
```

```yaml
Type: String
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: True
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Path

> Specifies the physical directory for the new Web site (in the virtual directories folder).

Reviewer: ```suggestion
Specifies the physical directory for the new website (in the virtual directories folder).
```

### -SecureSocketsLayer
Enables Secure Sockets Layer (SSL) encryption for this Web application.

> If you choose to use SSL, you must import a server certificate to SharePoint and assign it to the IIS web site for this web application.
> Until this is done, the Web application will be inaccessible from this IIS Web site.

Reviewer: ```suggestion
If you choose to use SSL, you must import a server certificate to SharePoint and assign it to the IIS website for this web application.
Until this is done, the web application will be inaccessible from this IIS website.
```

The default value is False.

> If this parameter is omitted or set to False, this Web application will use HTTP for the specified port.

Reviewer: ```suggestion
If this parameter is omitted or set to False, this web application will use HTTP for the specified port.
```

Accept wildcard characters: False
```

### -UseServerNameIndication

> Specifies that the Secure Sockets Layer (SSL) binding of this IIS web site should use Server Name Indication (SNI).
> Server Name Indication allows multiple IIS web sites with unique host headers and unique server certificates to share the same SSL port.
> If Server Name Indication isn't used, all IIS web sites sharing the same SSL port must share the same server certificate.

Reviewer: ```suggestion
Specifies that the Secure Sockets Layer (SSL) binding of this IIS website should use Server Name Indication (SNI).
Server Name Indication allows multiple IIS websites with unique host headers and unique server certificates to share the same SSL port.
If Server Name Indication isn't used, all IIS websites sharing the same SSL port must share the same server certificate.
```
